loader/windows/icd_windows_apppackage.cpp

```diff
@@ -46,6 +58,10 @@ extern "C" bool khrIcdOsVendorsEnumerateAppPackage()
         KHR_ICD_TRACE("Failed to init WinRT\n");
         return false;
     }
+    auto Cleanup = MakeScopeExit([]()
+        {
+            Windows::Foundation::Uninitialize();
```

Comment:
Should this be included in the init change?  I'm not clear if you sent separate commits just for ease of review (Thank you!) and plan to squash them, or if they'll go in separately.  It seems like init and unint should come as one piece though.

Reply:
I was planning to squash them together. I can go ahead and squash those together ahead-of-time, in case you'd prefer to keep these separate after merging.